src/scripts/main.js

```diff
@@ -1,11 +1,26 @@
 'use strict';
 
 function waitFor(element, eventName) {
-  // write your code here
+  return new Promise((resolve) => {
+    const eventHandler = (e) => {
+      element.removeEveventListener(eventName, eventHandler);
```

Comment:
There is a typo in the method name `removeEveventListener`. It should be `removeEventListener`.